bin/ldnode.js

```diff
@@ -97,6 +97,11 @@ var argv = require('nomnom')
     full: 'default-app',
     help: 'URI to use as a default app for resources (default: https://linkeddata.github.io/warp/#/list/)'
   })
+  .option('signup', {
```

Comment:
Can we rename this option to `singleUserMode` (from our in-person discussion / #201?)